plugins/plugin-chart-echarts/src/MixedTimeseries/transformProps.ts

```diff
@@ -282,14 +278,14 @@ export default function transformProps(
       appendToBody: true,
       trigger: richTooltip ? 'axis' : 'item',
       formatter: (params: any) => {
-        const value: number = !richTooltip ? params.value : params[0].value[0];
-        const prophetValue: any[] = !richTooltip ? [params] : params;
+        const xValue: number = richTooltip ? params[0].value[0] : params.value[0];
+        const prophetValue: any[] = richTooltip ? params : [params];
```

Comment:
When `richTooltip` is disabled, we were referencing the full `value`, which is always a tuple with x and y value. Here I'm renaming it to `xValue` to better reflect that it is in fact the x-value (the first element of the tuple) and to disambiguate it from the `value` parameter in the `forEach` loop a few lines below.